compiler/ghc/default.nix

```diff
@@ -274,7 +280,8 @@ let
           + lib.optionalString (!enableShared) "+no_dynamic_ghc"
           + lib.optionalString useLLVM "+llvm"
           + lib.optionalString enableDWARF "+debug_info"
-          + lib.optionalString targetPlatform.isGhcjs "+native_bignum+no_profiled_libs"
+          + lib.optionalString (enableNativeBignum || targetPlatform.isGhcjs) "+native_bignum"
```

Comment:
Also, this line seems to break the build for GHCs older than 9.6! It seems the `native_bignum` build flavour was only added recently :(